crates/jstz_node/src/lib.rs

```diff
@@ -66,9 +83,14 @@ fn router() -> OpenApiRouter<AppState> {
         .merge(OperationsService::router_with_openapi())
         .merge(AccountsService::router_with_openapi())
         .merge(LogsService::router_with_openapi())
+        .route("/ping", get(health_check))
```

Comment:
Maybe rename this route to something more obvious for health checks, e.g. `/health`?

And since this handler doesn't really do anything, I think something like this is sufficient:
```rs
.route("/health", get(http::StatusCode::OK))
```

Reply:
done!